cmd/spire-server/cli/validate/validate_posix_test.go

```diff
@@ -0,0 +1,259 @@
+//go:build !windows
+
+package validate
+
+import (
+	"bytes"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	common_cli "github.com/spiffe/spire/pkg/common/cli"
+	"github.com/spiffe/spire/pkg/common/fflag"
+	"github.com/stretchr/testify/require"
+)
+
+func serverConfigWithPlugins(pluginConfig string) string {
+	return `
+server {
+	trust_domain = "example.org"
+  data_dir = "/opt/spire/server/data"
+}
+
+plugins {
+` + pluginConfig + `
+}
+`
+}
+
+func TestValidateConfig(t *testing.T) {
+	configDir := t.TempDir()
```

Comment:
this is going to create a single folder for all tests, that is going to prevent us to run tests in parallel, can you move this one to be inside each testcase?